src/components/therapistProfile/textFieldWithEdited/index.tsx

```diff
@@ -0,0 +1,91 @@
+import TextField from '@mui/material/TextField';
+import React, { useState } from 'react';
+import IconButton from '@mui/material/IconButton';
+import InputAdornment from '@mui/material/InputAdornment';
+import Edit from '@mui/icons-material/Edit';
+import { Button } from '@mui/material';
+import { useSnackbar } from 'notistack';
+import getStyle from './style';
+import axiosInstance from '../../../utils/apis/axios';
+
+// const isEditable = true;
+
+interface Props {
+  value: string | number;
+  dataType: 'fullName' | 'bio' | 'hourlyRate' |'major';
+  // eslint-disable-next-line no-unused-vars
+  onChange: (event: React.ChangeEvent<HTMLInputElement>) => void;

```

Comment:
```suggestion
  onChange: ChangeEventHandler<HTMLInputElement | HTMLTextAreaElement>;

```
Can u please try to fix this in your components, as well as please move interfaces to a separate files